src/cdk/coercion/coercion.md

```diff
@@ -1,3 +1,59 @@
-### Coercion
-
 Utility functions for coercing `@Input`s into specific types.
+
+### Example
+
+```ts
```

Comment:
Should we make this a full example? It's pretty extensive for a code block

Reply:
My reasoning for keeping it inline is that the example won't be particularly visual since the coercion only affects the public API of the component.